Sources/System/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#[[
+This source file is part of the Swift System open source project
+
+Copyright (c) 2019 Apple Inc. and the Swift Numerics project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_library(System
+  Errno.swift
```

Comment:
I don't suppose there's any reasonable way to say `*.swift` within a  directory? SPM already dictates that our source code matches our targets

Reply:
No, that is the way that CMake is designed - to be explicit.  If you try to use a glob, any changes to the source tree structure will not be reflected in the build until a clean build is done.